backbone.js

```diff
@@ -400,7 +400,12 @@
     if (options.collection) this.collection = options.collection;
     if (options.parse) attrs = this.parse(attrs, options) || {};
     var defaults = _.result(this, 'defaults');
+
+    // Additional `_.defaults()` wrap after `_.extend()` makes sense
+    // when `attrs` has a property explicitly set to `undefined`.
+    // Discussion: https://github.com/jashkenas/backbone/issues/3842
```

Comment:
You can just use GitHub's issue shorthand `#1234`. You could use this as a model: https://github.com/jashkenas/backbone/blob/master/backbone.js#L1158-L1159